ui/src/components/header.tsx

```diff
@@ -270,17 +270,20 @@ export const Header = () => {
                 <span>{user.username}</span>
               </div>
             </DropdownMenuItem>
+            {user.hasRole(['superuser']) && (
```

Comment:
Commit message:
* Typo in "a bit" and "go first"
* "make clear that it is"
* "but about server"